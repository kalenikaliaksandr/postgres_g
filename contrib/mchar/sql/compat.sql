--- table based checks

select '<' || ch || '>', '<' || vch || '>' from chvch;
select * from chvch where vch = 'One space';
select * from chvch where vch = 'One space ';

select * from ch where chcol = 'abcd' order by chcol;
select * from ch t1 join ch t2 on t1.chcol = t2.chcol order by t1.chcol, t2.chcol;
select * from ch where chcol > 'abcd' and chcol<'ee';
select * from ch order by chcol;

