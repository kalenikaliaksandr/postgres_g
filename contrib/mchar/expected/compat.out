--- table based checks
select '<' || ch || '>', '<' || vch || '>' from chvch;
    ?column?    |   ?column?   
----------------+--------------
 <No spaces   > | <No spaces>
 <One space   > | <One space >
 <1 space     > | <1 space >
(3 rows)

select * from chvch where vch = 'One space';
      ch      |    vch     
--------------+------------
 One space    | One space 
(1 row)

select * from chvch where vch = 'One space ';
      ch      |    vch     
--------------+------------
 One space    | One space 
(1 row)

select * from ch where chcol = 'abcd' order by chcol;
              chcol               
----------------------------------
 abcd                            
 AbcD                            
(2 rows)

select * from ch t1 join ch t2 on t1.chcol = t2.chcol order by t1.chcol, t2.chcol;
              chcol               |              chcol               
----------------------------------+----------------------------------
 abcd                             | AbcD                            
 abcd                             | abcd                            
 AbcD                             | AbcD                            
 AbcD                             | abcd                            
 abcz                             | abcz                            
 defg                             | dEfg                            
 defg                             | defg                            
 dEfg                             | dEfg                            
 dEfg                             | defg                            
 ee                               | Ee                              
 ee                               | ee                              
 Ee                               | Ee                              
 Ee                               | ee                              
(13 rows)

select * from ch where chcol > 'abcd' and chcol<'ee';
              chcol               
----------------------------------
 abcz                            
 defg                            
 dEfg                            
(3 rows)

select * from ch order by chcol;
              chcol               
----------------------------------
 abcd                            
 AbcD                            
 abcz                            
 defg                            
 dEfg                            
 ee                              
 Ee                              
(7 rows)

